apps/frontend/src/app/5_pages/RewardsPage/hooks/useGetLiquidSovClaimAmount.ts

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useAccount } from '../../../../hooks/useAccount';
+import { useGetProtocolContract } from '../../../../hooks/useGetContract';
+import { decimalic } from '../../../../utils/math';
+
+export const useGetLiquidSovClaimAmount = () => {
```

Comment:
The UI shows me I have `0 SOV` liquid but withdraw button is enabled. Why is that? 
<img width="717" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/b17440e1-c275-438b-b827-c682c8a9360a">